pyproject.toml

```diff
@@ -5,6 +5,14 @@ authors = [
   { name="Jiri Janos", email="jiri.janos98@gmail.com" },
   { name="Daniel Hollas", email="danekhollas@gmail.com" },
 ]
+license = {file = 'LICENSE'}
+classifiers = [
+  'Development Status :: 5 - Production/Stable',
+  'License :: OSI Approved :: MIT License',
+  'Programming Language :: Python',
```

Comment:
Just out of curiousity, why is there `'Programming Language :: Python'` Python twice? I see one is for Python3 but does it make any difference?

Reply:
In practice this will not make a difference. We're just saying that we do not support Python2